integration-tests/src/main/java/eu/tsystems/mms/tic/testframework/core/utils/Log4jFileReader.java

```diff
@@ -140,10 +139,14 @@ public String filterLogForFollowingEntry(final String searchString) {
     public void assertTestStatusPerMethod(final String classNameSlug, final String methodNameSlug, final Status expectedStatus) {
         final List<String> foundEntries = filterLogForTestMethod(classNameSlug, methodNameSlug);
 
-        for (final String entry : foundEntries) {
-            log().debug("Asserting '{}' for '{}'", expectedStatus, methodNameSlug);
-            Assert.assertTrue(entry.contains(expectedStatus.title), String.format("'%s' has status '%s'", methodNameSlug, expectedStatus));
-        }
+        log().debug("Asserting '{}' for '{}'", expectedStatus, methodNameSlug);
+        foundEntries.stream()
+                .filter(line -> line.contains(expectedStatus.title))
+                .findFirst()
```

Comment:
Would not `anyMatch` work just as well here? Or _must_ we discard results after the first?
(`value` is unused, so we do not actually need a `find` operation.)

Reply:
It does not work without `value`: `() -> log().info(...)` --> IntelliJ: Cannot infer functional interface type

Reply:
And it's just a helper class for tests. ;-)

Reply:
I mean like this:
```java
if (foundEntries.stream().anyMatch(line -> line.contains(expectedStatus.title))) {
  log().info("Found status {}", expectedStatus.title);
}
else {
  Assert.fail(String.format("'%s' should have status '%s'", methodNameSlug, expectedStatus));
}
```
?
`value` is unused in `ifPresentOrElse`, so we do not actually need a result item.

Reply:
(Or alternatively: Did you forget to output `value` in the `ifPresent` part?)

Reply:
Because of unique combination of class and method I have one result or nothing.

And value is not needed in output, I dont want to log the whole line. But value must be declared.